packages/client/src/__tests__/integration/happy/signals/test.ts

```diff
@@ -4,9 +4,16 @@ import execa from 'execa'
 import path from 'path'
 import { EXIT_MESSAGE, READY_MESSAGE } from './__helpers__/constants'
 
+const testIf = (condition: boolean) => (condition ? test : test.skip)
+
+const userSignalsSupported = process.platform !== 'win32'
```

Comment:
What is the story here?

Reply:
SIGUSR1 and SIGUSR2 don't exist on Windows.